python/oneflow/framework/env_util.py

```diff
@@ -30,11 +30,23 @@
 from oneflow import oneflow_deprecate
 
 
-def api_all_device_placement(device_type: str) -> None:
-    """Return a placement containing all devices of all machines under env.
+def api_all_device_placement(device_type: str) -> oneflow._oneflow_internal.placement:
+    r"""
+    Return a placement containing all devices of all machines under env.
```

Comment:
```suggestion
    Return a placement containing all devices of all machines in current cluster.
```
这样是不是准确点？我不确认，以你自己为准

Reply:
这里不应该是集群，集群是物理硬件相关的概念，这的all devices指的是我们初始化的环境中所有设备，和初始化参数有关